src/components/myProfile/myProfile.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import Table from 'react-bootstrap/Table';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Container from 'react-bootstrap/Container';
+
+function MyProfil() {
```

Comment:
@aubinleyoung  I recommend you try to avoid typos in your naming conventions MyProfile instead of MyProfil.